### PR TITLE
fix to camel case for new Image().crossOrigin

### LINKED ISSUE #227: Images with CORS policy
Occasionally, we need to load images from domains with a CORS policy. This means setting the `crossorigin` attribute on the `img` tag (and on an `Image` object when loading from JavaScript).

Currently `react-image` doesn't allow you to specify this on the `Image` used to load images, so it simply fails. Could we create a `crossorigin` prop on `Img`?

e.g.

```js
const myComponent = () => (
  <Img src={'https://www.example.com/foo.jpg'} crossorigin="anonymous" />
)
```

### PR DESCRIPTION
**Pull request by jharris4**
Fixes #227

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -132,8 +133,9 @@ class Img extends Component {
     }
     this.i.src = this.sourceList[this.state.currentIndex]
 
-    if (this.props.crossorigin) {
-      this.i.crossorigin = this.props.crossorigin;
+    if (this.props.crossOrigin || this.props.crossorigin) {
+      this.i.crossOrigin = this.props.crossOrigin || this.props.crossorigin
+      this.i.crossorigin = this.props.crossOrigin || this.props.crossorigin
```
Comment (mbrevda) [inline src/index.js]: Do we need to set crossorigin twice (i.e. both lowercase and camelcase)?

**Comment by mbrevda** [comment]
Do you think that `crossorigin` should NOT be excluded so that it ends up on the final `` element? Additionally, should we set ONLY `crossOrigin` on the `Image()` object?

File: src/index.js (hunk 1 above)
Comment (jharris4) [reply src/index.js]: I wasn't sure, so I just set both camel & lower case. It doesn't do any harm, but we could remove the second one I think.

**Comment by jharris4** [comment]
> Do you think that `crossorigin` should NOT be excluded so that it ends up on the final `` element? Additionally, should we set ONLY `crossOrigin` on the `Image()` object?

Good question. I tested in Chrome, and setting the `crossOrigin` on the `Image` worked just fine without setting the attribute/prop on the `img` element.

If we do pass the prop to the ``, then I think we should be careful about what the name of the prop is. I think it should **always** be camelCase.

For example, the `leaflet` package that I'm using in the same project as `react-image` does it like [this](https://github.com/Leaflet/Leaflet/blob/master/src/layer/tile/TileLayer.js#L143).

**Comment by mbrevda** [comment]
Is your understanding, based on the docs linked above, that the js `Image()` uses camelClass, while the html ``uses lower class?

**Comment by jharris4** [comment]
That was my general understanding, but the `leaflet` code I linked to above does this:

```
var tile = document.createElement('img');
tile.crossOrigin = this.options.crossOrigin === true ? '' : this.options.crossOrigin;
```

which implies that camelCase can (should?) be used even for `` tags.

The thing I do know, is that changing to camel-case is necessary for me to be able to use `react-image` with images served over `https`...

**Pull request closed without merging**